diff --git a/node_modules/gatsby-plugin-image/dist/components/hooks.js b/node_modules/gatsby-plugin-image/dist/components/hooks.js
index 6148dc4..a3a91bd 100644
--- a/node_modules/gatsby-plugin-image/dist/components/hooks.js
+++ b/node_modules/gatsby-plugin-image/dist/components/hooks.js
@@ -148,7 +148,7 @@ function getMainProps(isLoading, isLoaded, images, loading, style) {
     if (style === void 0) { style = {}; }
     // fallback when it's not configured in gatsby-config.
     if (!gatsbyImageIsInstalled()) {
-        style = __assign({ height: "100%", left: 0, position: "absolute", top: 0, transform: "translateZ(0)", transition: "opacity 250ms linear", width: "100%", willChange: "opacity" }, style);
+        style = __assign({ height: "100%", left: '0px', position: "absolute", top: '0px', transform: "translateZ(0px)", transition: "opacity 250ms linear", width: "100%", willChange: "opacity" }, style);
     }
     var result = __assign(__assign({}, images), { loading: loading, shouldLoad: isLoading, "data-main-image": "", style: __assign(__assign({}, style), { opacity: isLoaded ? 1 : 0 }) });
     return result;
@@ -166,17 +166,17 @@ function getPlaceholderProps(placeholder, isLoaded, layout, width, height, backg
         }
         else if (layout === "constrained") {
             wrapperStyle.position = "absolute";
-            wrapperStyle.top = 0;
-            wrapperStyle.left = 0;
-            wrapperStyle.bottom = 0;
-            wrapperStyle.right = 0;
+            wrapperStyle.top = '0px';
+            wrapperStyle.left = '0px';
+            wrapperStyle.bottom = '0px';
+            wrapperStyle.right = '0px';
         }
         else if (layout === "fullWidth") {
             wrapperStyle.position = "absolute";
-            wrapperStyle.top = 0;
-            wrapperStyle.left = 0;
-            wrapperStyle.bottom = 0;
-            wrapperStyle.right = 0;
+            wrapperStyle.top = '0px';
+            wrapperStyle.left = '0px';
+            wrapperStyle.bottom = '0px';
+            wrapperStyle.right = '0px';
         }
     }
     if (objectFit) {
@@ -190,9 +190,9 @@ function getPlaceholderProps(placeholder, isLoaded, layout, width, height, backg
     if (!gatsbyImageIsInstalled()) {
         result.style = {
             height: "100%",
-            left: 0,
+            left: '0px',
             position: "absolute",
-            top: 0,
+            top: '0px',
             width: "100%",
         };
     }
diff --git a/node_modules/gatsby-plugin-image/dist/components/layout-wrapper.js b/node_modules/gatsby-plugin-image/dist/components/layout-wrapper.js
index 7f0eaf1..469ddd1 100644
--- a/node_modules/gatsby-plugin-image/dist/components/layout-wrapper.js
+++ b/node_modules/gatsby-plugin-image/dist/components/layout-wrapper.js
@@ -64,7 +64,7 @@ function getSizer(layout, width, height) {
         sizer = "<div aria-hidden=\"true\" style=\"padding-top: ".concat((height / width) * 100, "%;\"></div>");
     }
     if (layout === "constrained") {
-        sizer = "<div style=\"max-width: ".concat(width, "px; display: block;\"><img alt=\"\" role=\"presentation\" aria-hidden=\"true\" src=\"data:image/svg+xml;charset=utf-8,%3Csvg%20height='").concat(height, "'%20width='").concat(width, "'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E\" style=\"max-width: 100%; display: block; position: static;\"></div>");
+        sizer = "<div style=\"max-width: ".concat(width,`${width > 0 ? 'px' : ''}`, "; display: block;\"><img alt=\"\" role=\"presentation\" aria-hidden=\"true\" src=\"data:image/svg+xml;charset=utf-8,%3Csvg%20height='").concat(height, "'%20width='").concat(width, "'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E\" style=\"max-width: 100%; display: block; position: static;\"></div>");
     }
     return sizer;
 }
